test/e2e/nodepool_prev_release_test.go

```diff
@@ -4,26 +4,29 @@
 package e2e
 
 import (
+	"testing"
+
 	hyperv1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
 	e2eutil "github.com/openshift/hypershift/test/e2e/util"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilrand "k8s.io/apimachinery/pkg/util/rand"
-	"testing"
 )
 
 type NodePoolPrevReleaseCreateTest struct {
 	DummyInfraSetup
-	hostedCluster *hyperv1.HostedCluster
-	release       string
-	clusterOpts   e2eutil.PlatformAgnosticOptions
+	hostedCluster     *hyperv1.HostedCluster
+	release           string
+	clusterOpts       e2eutil.PlatformAgnosticOptions
+	expectVersionSkew bool
```

Comment:
should this be named supportedVersionSkew=true/false
or expectedSupportedVersionSkew=true/false
?